app/src/renderer/scripts/wallet.js

```diff
@@ -0,0 +1,183 @@
+const bip39 = require(`bip39`)
+const bip32 = require(`bip32`)
+const bech32 = require(`bech32`)
+const secp256k1 = require(`secp256k1`)
+import sha256 from "crypto-js/sha256"
+import ripemd160 from "crypto-js/ripemd160"
+import CryptoJS from "crypto-js"
+
```

Comment:
maybe we can add some comment to explain the pieces of this string

Reply:
https://github.com/cosmos/voyager/issues/1891